### PR TITLE
🔥 Remove support for password authentication

### PR DESCRIPTION
**Pull request by psbanka**
Fastly' API no longer supports authentication using passwords. Removing this functionality from the library.

Co-authored-by: sshahlapour@fastly.com <Shadi Shahlapour>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.0] - September 30, 2020
```
Comment (annawinkler) [inline CHANGELOG.md:5]: I'm curious if this should be a major version bump if it's a breaking change? Based on my understanding of semver, I would think we'd bump this to `1.0.0`. What are your thoughts?

**Comment by annawinkler** [comment]
🥳 Thank you for this PR! I have one question about the versioning (see the comment).

File: CHANGELOG.md (hunk 1 above)
Comment (psbanka) [reply CHANGELOG.md:5]: According to [the semver spec](https://semver.org/#spec-item-4), when a release is pre 1.0, then 0.x.y changes can be considered breaking.

Comment (annawinkler) [reply CHANGELOG.md:5]: Ah - ok that's a great point.

**Review by annawinkler — APPROVED** [review]
🥇 Thank you!

**Pull request merged**